README.yaml

```diff
@@ -83,40 +67,63 @@ usage: |-
   (which tests and deploys the example on AWS), see [test](test).
 
   ```hcl
-  # Create a standard label resource. See [null-label](https://github.com/cloudposse/terraform-null-label/#terraform-null-label--)
-  module "label" {
-    source  = "cloudposse/label/null"
-    # Cloud Posse recommends pinning every module to a specific version, though usually you want to use the current one
-    # version = "x.x.x"
+  locals {
+    enabled = module.this.enabled
+
+    additional_allowed_roles = compact([for prometheus in module.prometheus : prometheus.outputs.access_role_arn])
+  }
+
+  module "security_group" {
+    source  = "cloudposse/security-group/aws"
+    version = "2.2.0"
 
-    namespace = "eg"
-    name      = "example"
+    enabled = local.enabled && var.private_network_access_enabled
+
+    allow_all_egress = true
+    rules            = []
+    vpc_id           = module.vpc.outputs.vpc_id
+
+    context = module.this.context
   }
 
-  module "example" {
-    source  = "cloudposse/*****/aws"
+  module "managed_grafana" {
+    source  = "cloudposse/managed-grafana/aws"
     # Cloud Posse recommends pinning every module to a specific version
     # version = "x.x.x"
 
-    example = "Hello world!"
+    enabled = local.enabled
+
+    prometheus_policy_enabled = var.prometheus_policy_enabled
+    additional_allowed_roles  = local.additional_allowed_roles
 
-    context = module.label.this
+    vpc_configuration = var.private_network_access_enabled ? {
+      subnet_ids         = module.vpc.outputs.private_subnet_ids
+      security_group_ids = [module.security_group.id]
+    } : {}
+
+    context = module.this.context
+  }
+
+  resource "aws_grafana_role_association" "sso" {
+    for_each = local.enabled ? {
+      for association in var.sso_role_associations : association.role => association
+    } : {}
+
+    role      = each.value.role
+    group_ids = each.value.group_ids
+
+    workspace_id = module.managed_grafana.workspace_id
   }
```

Comment:
Why not support this in the module directly? It can be optional.

Reply:
okay I'll add it

Reply:
added